.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: Report a problem in the training materials.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to report a problem in the training materials.
+              Note: If you are learning Nextflow and need help with the content, ask your questions in the [Seqera community forum](https://community.seqera.io/c/help/37).
+    - type: input
+      id: version
+      attributes:
+          label: Training materials version
+          description: What's the git commit hash of the training materials you are using?
+          placeholder: ex. email@example.com
+      validations:
+          required: false
+    - type: textarea
+      id: whats-wrong
+      attributes:
+          label: What's wrong?
+          description: What's the problem in the training materials?
```

Comment:
This seems a bit redundant with the label, maybe state the purpose, ie are we looking for a high level summary here?